src/modules/tracker.js

```diff
@@ -108,6 +127,33 @@ class Tracker {
     return true;
   }
 
+  /**
+   * Send input focus event to API
+   * @private
+   * @function trackInputFocusV2
+   * @param {string} user_input - Input at the time user focused on the search bar
+   * @param {object} [networkParameters] - Parameters relevant to the network request
+   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
+   * @returns {(true|Error)}
+   * @description User focused on search input element
+   * @example
+   * constructorio.tracker.trackInputFocusV2("text");
+   */
+  trackInputFocusV2(user_input = '', networkParameters = {}) {
+    const baseUrl = 'https://ac.cnstrc.com/v2/behavioral_action/focus?';
+    const bodyParams = { user_input };
+
+    const requestMethod = 'POST';
+    const requestBody = applyParams(bodyParams, {
+      ...this.options,
+      requestMethod,
+    });
+    this.requests.queue(`${baseUrl}${applyParamsAsString({}, this.options)}`, requestMethod, requestBody, networkParameters);
```

Comment:
Not specific to this function, more of a question about all V2 endpoints. Do we still have to duplicate params in body of the POST and in the query params?

Reply:
Don't think so. The docs have query params, json params separate and there is no overlap.

https://docs.constructor.io/rest_api/behavioral_logging/autocomplete/